app/services/manage_action.rb

```diff
@@ -1,91 +1,85 @@
 # frozen_string_literal: true
 
 class ManageAction
-  attr_reader :params
-
   def self.create(params, extra_params: {}, skip_queue: false, skip_counter: false)
     new(params, extra_params: extra_params, skip_queue: skip_queue, skip_counter: skip_counter).create
   end
 
   def initialize(params, extra_params: {}, skip_queue: false, skip_counter: false)
-    @params = params
+    @params = params.clone
     @skip_queue = skip_queue
     @skip_counter = skip_counter
-    @extra_attrs = extra_params
+    @extra_attrs = extra_params.clone
+    sanitize_params
```

Comment:
Can you give this method a name that's more clear about what it does?

Reply:
`sanitize_params` refers to preparing and formatting the params to be used by the service. I've seen this word used widely in Ruby for this. We're also using it in a few other places in Champaign. But I'm happy to change it to something else if it's not clear, any ideas?